crmsh/bootstrap.py

```diff
@@ -422,7 +422,6 @@ def _get_sbd_device_interactive(self):
         Get sbd device on interactive mode
         """
         if _context.yes_to_all:
-            warn("Not configuring SBD ({} left untouched).".format(SYSCONFIG_SBD))
```

Comment:
Instead of delete, prabably makes sense to add one more line as below.
```
warn("Not configuring SBD - STONITH will be disabled.")
```

Reply:
Changed